frontend/packages/kubevirt-plugin/src/constants/vm/storage/disk-bus.ts

```diff
@@ -8,21 +8,21 @@ export class DiskBus extends SelectDropdownObjectEnum<string> {
     labelKey: 'kubevirt-plugin~virtio',
     // t('kubevirt-plugin~Optimized for best performance. Supported by most Linux distributions. Windows requires additional drivers to use this model')
     descriptionKey:
-      'kubevirt-plugin~Optimized for best performance. Supported by most Linux distributions. Windows requires additional drivers to use this model',
+      'kubevirt-plugin~Hotplug is enabled only for "scsi" interface. Optimized for best performance. Supported by most Linux distributions. Windows requires additional drivers to use this model',
```

Comment:
the "not supporting hotplug" message should be at the end of the description, 
you are describing what is "virtio" the most important features of this options is that it's' optimized for performance, the fact that it's not supported by hotplug is scondary and should be at the end of the description

Reply:
applied